frontend/src/components/shared/VirtualDropdown/VirtualDropdown.tsx

```diff
@@ -25,17 +27,11 @@ function FixedSizeListItem(props: FixedSizeListeItemProps): ReactElement {
   return (
     <StyledDropdownListItem
       key={item.value}
-      style={{
-        boxSizing: "border-box",
-        paddingTop: 0,
-        paddingBottom: 0,
-        display: "flex",
-        alignItems: "center",
-        ...style,
-      }}
+      className="dropdownListItem"
+      style={style}
       {...restChildProps}
     >
-      {item.label}
+      <span className="noTextOverflow">{item.label}</span>
```

Comment:
Not a blocker but curious why not move to CSS in JS?

Reply:
CSS will be easier to copy to emotion.